steam-resources/steam_language_parser/parser/token_analyzer.js

```diff
@@ -20,21 +21,21 @@ exports.EnumNode = function() {
 
 exports.analyze = function(tokens) {
   var root = new exports.Node();
-  
+
   while (tokens.length > 0) {
     var cur = tokens.shift();
-    
+
     switch (cur.name) {
       case 'EOF':
         break;
       case 'preprocess':
         var text = expect(tokens, 'string');
-        
+
         if (cur.value == 'import') {
-          var parentTokens = language_parser.tokenizeString(require('fs').readFileSync(text.value, { encoding: 'ascii' }));
-          
+          var parentTokens = language_parser.tokenizeString(file_cache.cache[text.value]);
```

Comment:
Might be a good idea to throw here in case `file_cache.cache[text.value]` is `undefined`.